crates/db_schema/src/impls/comment.rs

```diff
@@ -187,11 +191,8 @@ impl Crud for Comment {
     diesel::delete(comment.find(comment_id)).execute(conn)
   }
 
-  fn create(conn: &PgConnection, comment_form: &CommentForm) -> Result<Self, Error> {
-    use crate::schema::comment::dsl::*;
-    insert_into(comment)
-      .values(comment_form)
-      .get_result::<Self>(conn)
+  fn create(_conn: &PgConnection, _comment_form: &CommentForm) -> Result<Self, Error> {
+    unimplemented!();
```

Comment:
Add a comment pointing to other create/upsert method.

Reply:
Can do. Just want to note that unimplemented is pretty scary unless you override the trait with a struct impl. I removed all the code and cargo check passed fine... apparently unimplemented just runs a panic, doesn't do any compile-time checking.